packages/injectable/core/src/dependency-injection-container/privateInjectFor.js

```diff
@@ -83,6 +83,14 @@ const getInstance = ({
   const minimalDi = {
     inject: minimalInject,
 
+    injectWithMeta: (alias, parameter) =>
+      di.injectWithMeta(
+        alias,
+        parameter,
+        newContext,
+        injectableToBeInstantiated,
```

Comment:
Untested code.

Reply:
Which part? I added a unit test to make sure that `injectWithMeta` was available from the `di` parameter of `instantiate`